maven-embedder/src/main/java/org/apache/maven/cli/CLIReportingUtils.java

```diff
@@ -136,6 +136,13 @@ static Properties getBuildProperties() {
     }
 
     public static void showError(Logger logger, String message, Throwable e, boolean showStackTrace) {
+        if (logger == null) {
```

Comment:
How can the logger be null here?

Reply:
If `showError` is called before the logger is initialised...